src/utils/safe.ts

```diff
@@ -0,0 +1,9 @@
+import { SafeTransactionType } from 'components/v2/V2Project/V2ProjectSafeDashboard/V2ProjectSafeDashboard'
```

Comment:
**suggestion** WDYT about moving this file to `lib/`?

I think lib is a better spot for service-specific helpers (like safe, snapshot etc.)

Reply:
Good call